openshift2nulecule/utils.py

```diff
@@ -64,7 +64,25 @@ def get_path(path):
             return os.path.abspath(expanded_path)
 
 
-def generate_dockerfile(nulecule_dir):
+def generate_dockerfile(nulecule_dir, custom_atomicapp_version):
+    """
+    Generate a Dockerfile for an exported application automatically, by reading
+    the contents of the directory where the exported artifacts reside.
+
+    Args:
+        nulecule_dir (str): path to the directory where all Nulecule artifacts
+                            reside
+        custom_atomicapp_version (str): User can specify the Atomic App version
+                                        different from ATOMICAPP_VERSION in
+                                        constants file.
+
+    Returns:
+        None
+    """
+    if custom_atomicapp_version:
```

Comment:
Wouldn't be better to add default option to arg parser, instead of this `if`?

```
self.parser.add_argument("--atomicapp-ver",
    help="Specify custom Atomic App version for the Dockerfile that will be generated.",
    type=str,
    default=ATOMICAPP_VERSION,
    required=False)
```